index.html

```diff
@@ -29,7 +29,7 @@ <h4 class="input-title header-text">Title</h4>
           <input class="title-input" type="text" name="" value="">
           <h4 class="input-body header-text">Body</h4>
           <input class="body-input" type="text" name="" value="">
-          <button class="save-button header-text" type="button" name="button">Save</button>
+          <button disabled class="disable-style save-button  header-text" type="button" name="button">Save</button>
```

Comment:
I would like to go ove rthis later to make sure I know what that "disabled" piece is, as far as terms of art goes.